packages/tofu/src/findGlobals.js

```diff
@@ -3,26 +3,36 @@
 const { default: traverse } = require('@babel/traverse')
 const { isInFunctionDeclaration, isMemberLikeExpression } = require('./util')
 
-const nonReferenceIdentifiers = [
+/**
+ * Types which are not references to globals
+ */
+const nonReferenceIdentifiers = new Set([
+  'ArrayPatten',
+  'BreakStatement',
+  'CatchClause',
+  'ClassMethod',
+  'ContinueStatement',
+  'ExportDefaultSpecifier',
+  'ExportSpecifier',
   'FunctionDeclaration',
   'FunctionExpression',
-  'ClassMethod',
+  'ImportDefaultSpecifier',
+  'ImportNamespaceSpecifier',
+  'ImportSpecifier',
   'LabeledStatement',
-  'BreakStatement',
-  'ContinueStatement',
-  'CatchClause',
-  'ArrayPatten',
+  'MetaProperty',
   'RestElement',
```

Comment:
I hope that's just a reorder. Don't have the patience to verify :facepalm:

Reply:
yes, but `MetaProperty` is new, and refers to `import.meta`